handlers/runCmd.ts

```diff
@@ -81,16 +83,37 @@ export const RunCmd = async (
 
   denoCmd.push(appFile);
 
-  let appStatus = await Deno.run({
-    cmd: denoCmd,
-  }).status();
+  // * reload project when files changes
+  const runApp = (): Deno.Process => {
+    return Deno.run({
+      cmd: denoCmd,
+    });
+  };
+
+  let task = runApp();
+
+  let throttle = 200;
+
+  let timeout: number | null = null;
+
+  for await (const event of Deno.watchFs(".")) {
+    if (event.kind !== "access") {
+      if (timeout) clearTimeout(timeout);
+
+      timeout = setTimeout(() => {
+        task && task.close();
+        console.clear();
+        task = runApp();
```

Comment:
this part stops the process and restarts it, I proved it several times, although I am thinking of adding unit tests for more security

Reply:
@andreespirela `process.kill(process.pid)` can also be used but this is not available in Windows yet

Reply:
Yes, let's do that. After that's done, remember to update the documentation in the main repo. Branch off develop, and update the docs of the CLI and add docs to how to "watch" with Mandarine's CLI.

Note that this feature will be temporary: https://github.com/denoland/deno/pull/5955 . Once that lands in Deno we'll remove it

Reply:
the `--watch` flag will be present in deno 1.4 so it will be temporary